### PR TITLE
fix: sidebar logo not aligned with different version length

### PR DESCRIPTION
**Pull request by rbreeze**
With a different version length, sidebar logo was misaligned.

Before:
CleanShot 2023-02-08 at 08 49 43@2x

After:
CleanShot 2023-02-08 at 08 51 58@2x

Signed-off-by: Remington Breeze <remington@breeze.software>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Review by alexef — APPROVED** [review]
not a CSS expert, but I've tested the changes on Safari and Chrome and they look good!

**Review by alexmt — APPROVED** [review]
LGTM!

**Review by rpelczar — APPROVED** [review]
LGTM!

File: ui/src/app/sidebar/sidebar.scss [hunk 1]
```diff
@@ -51,7 +52,7 @@ $deselected-text: #818d94;
     &__version {
         cursor: pointer;
         width: 100%;
-        text-align: center;
+        text-align: left;
```
Comment (keithchong) [inline ui/src/app/sidebar/sidebar.scss:55]: I noticed that if the version is even longer, the collapsed button will get pushed to the right and will not be in view.

Try changing the css for version to this:

    &__version {
        cursor: pointer;
        width: 110px;
        text-align: left;
        white-space: nowrap;
        overflow: hidden;
        text-overflow: ellipsis;
    }

and see if it is ok with you.

Everything else LGTM

Comment (rbreeze) [reply ui/src/app/sidebar/sidebar.scss:55]: great suggestion, thank you!

**Pull request merged**